packages/fxa-auth-server/scripts/populate-vat-taxes.ts

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+import { ACTIVE_SUBSCRIPTION_STATUSES } from 'fxa-shared/subscriptions/stripe';
```

Comment:
I'm uncertain if this import is going to work.  Maybe change to a relative import?

Reply:
It seemed to work every time I ran it.